helix-sitemap.yaml

```diff
@@ -2,7 +2,7 @@ version: 1
 
 sitemaps:
   website:
-    origin: https://helpx-internal.stage.adobe.com
+    origin: https://helpx-internal.adobe.com
```

Comment:
Please revert this change

Reply:
I ended up using the same branch for both PRs. Shall I revert it after #41 is merged?